### ISSUE TITLE
Unexpected Argument Filesystem Issue

### ISSUE THREAD

**Issue by gyalowitzdtlr** [terraform-providers/terraform-provider-ignition#71]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
![image](https://user-images.githubusercontent.com/59739698/100122492-70812680-2e47-11eb-81a5-0c1658dcb727.png)

### Affected Resource(s)
Please list the resources as a list, for example:

Ignition
Vsphere

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```
![image](https://user-images.githubusercontent.com/59739698/100122674-a6bea600-2e47-11eb-9e4d-1994ac064384.png)

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?
Ignition configs generate
### Actual Behavior
What actually happened?
![image](https://user-images.githubusercontent.com/59739698/100123254-4c721500-2e48-11eb-8fd6-9280c519d559.png)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan -out`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?
This was working without issue until I updated to the most recent ignition.

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by gyalowitzdtlr** [comment]
From what I can see filesystem is part of these modules.  Basically each data source is now showing as an error.  Apologies up front if I am missing something simple.  As always, thanks in advance for your time.

**Comment by bgilbert** [comment]
You're using the [community-terraform-providers fork](https://github.com/community-terraform-providers/terraform-provider-ignition), not this project.  That fork supports Ignition spec v3.x, not v2.x, and spec v3 removes the `filesystem` field.

**Comment by gyalowitzdtlr** [comment]
Definitely confusing....thank you.

**Comment by gyalowitzdtlr** [comment]
Any idea how to specify this project in 13 without locally building it?

**Comment by gyalowitzdtlr** [comment]
Confusing, thank you.  So using the ignition documentation, I should be able to fix my module conforming to 3.x even though none of it is reflected in the Terraform documentation?

**Comment by bgilbert** [comment]
Yes.  The correct repo depends on what Linux distribution you're using though.  If you're on Fedora CoreOS, you'll need the fork and spec 3.x.  If you're using Flatcar Container Linux, you'll need to stay on this repo and spec 2.x.

**Comment by gyalowitzdtlr** [comment]
So I would have to build this repo locally then or is it still possible to pull this through terraform?

**Comment by bgilbert** [comment]
I'm not sure; I'm not actually a Terraform user.

**Comment by gyalowitzdtlr** [comment]
Thank you, I appreciate your help on this.
